app/models/rental.rb

```diff
@@ -40,12 +46,24 @@ def renter_is_assignable
   scope :inactive_rentals, -> { where(rental_status: %w(canceled dropped_off)) }
   scope :rented_by, ->(user) { where(renter_id: user) }
   scope :created_by, ->(user) { where(creator_id: user) }
+
+  #   if these queries ever become a problem this would be faster, i think a cross apply or something like that would probably be even faster.
+  #   select * from rentals where
+  #       ((select sum(amount) from financial_transactions where rental_id=rentals.id and
+  #         (transactable_type='Payment' or transactable_type='Cancelation'))
+  #       -(select sum(amount) from financial_transactions where rental_id=rentals.id and
+  #         not (transactable_type='Payment' or transactable_type='Cancelation')))
+  #       > 0
   scope :with_balance_due, -> { Rental.where id: Rental.select { |rental| rental.balance.positive? }.collect(&:id) }
-  scope :with_balance_over, ->(min) { Rental.where id: Rental.select { |rental| rental.balance >= min }.collect(&:id) }
+  scope :with_balance_over, ->(min) { Rental.where id: Rental.select { |rental| rental.balance > min }.collect(&:id) }
 
   delegate :payments, to: :financial_transactions
   delegate :department, to: :renter
 
+  def reservations
+    rentals_items.collect(&:reservation_id)
+  end; alias reservation_ids reservations
```

Comment:
Why write this on the same line as the `end`?

Reply:
https://github.com/umts/aggressive-epsilon/commit/cda7c968cac6d67d679a1e8fb490b8d0c52fa18a

I figured it was the right way to do things

Reply:
Hmm, Rubocop must've suggested it. Carry on.